### DISCUSSION TITLE
发票

### DISCUSSION THREAD

**Discussion by fangyalong** [community/community#75482] [New to GitHub]
github 充值如何开发票

**Comment by queenofcorgis** [comment]
Thanks for participating!
We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. What this means is that, until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
